File: Python/PGS/LEVEL2/구명보트/gunwoo.py
def solution(people, limit):
    s = 0
    l = len(people)-1
    people = sorted(people)
    answer = 0
    while s <= l:
        if people[s] + people[l] <= limit:
            s += 1
        l -= 1
        answer += 1
        
    return answer

File: Algorithm/PGS/LEVEL0/제곱수 판별하기/gunwoo.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        double sqrtn = Math.sqrt(n);
            
        if(sqrtn == (int) sqrtn){
            answer = 1;
        }
        else{
            answer = 2;
        }
        return answer;
    }
}